src/Fixer/ControlStructure/TrailingCommaInMultilineFixer.php

```diff
@@ -130,22 +132,26 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
 
     protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     {
-        $fixArrays = \in_array(self::ELEMENTS_ARRAYS, $this->configuration['elements'], true);
-        $fixArguments = \in_array(self::ELEMENTS_ARGUMENTS, $this->configuration['elements'], true);
-        $fixParameters = \PHP_VERSION_ID >= 8_00_00 && \in_array(self::ELEMENTS_PARAMETERS, $this->configuration['elements'], true); // @TODO: drop condition when PHP 8.0+ is required
-        $fixMatch = \PHP_VERSION_ID >= 8_00_00 && \in_array(self::MATCH_EXPRESSIONS, $this->configuration['elements'], true); // @TODO: drop condition when PHP 8.0+ is required
+        $configuredElements = $this->configuration['elements'];
```

Comment:
`diff` looks rather bad, however it is just the extraction `$configuredElements = $this->configuration['elements'];` to reduce the complexity when trying to read the code, and than simplify the checks below it.